.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Enhancement Request
+about: Suggest an enhancement to the test-infra project
```

Comment:
nit: There isn't really a "test-infra project" this repo is more of a collection of tools.
"Suggest an enhancement to the test-infra tooling." might be more appropriate.